src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -3412,6 +3440,8 @@ public struct ShutdownParams
         /// 
         public bool PerformShutdownSequence(ShutdownParams shutdownParams)
         {
+            Dispatcher.CurrentDispatcher.UnhandledException -= CurrentDispatcher_UnhandledException;
```

Comment:
we should make sure we unsubscribe when the Dynamo windows is closed in Revit, so that we do not catch non-Dynamo exceptions with Dynamo closed

Reply:
Will check.